tests/integrations/raftstore/test_multi.rs

```diff
@@ -351,11 +352,14 @@ fn test_leader_change_with_uncommitted_log<T: Simulator>(cluster: &mut Cluster<T
     ));
     // Make peer 2 have no way to know the uncommitted entries can be applied
     // when it's still follower.
-    cluster.add_send_filter(CloneFilterFactory(
-        RegionPacketFilter::new(1, 2)
-            .msg_type(MessageType::MsgHeartbeat)
-            .direction(Direction::Recv),
-    ));
+    cluster.add_send_filter(CloneFilterFactory(MessageCorruptFilter(
```

Comment:
Why touch this test?

Reply:
The test case blocks heartbeats from leader to a follower, and then transfers leadership to that peer. This behavior is broken by the PR.